File: Listas de Exercícios (Python)/Python/Lista 1 - Casa/5.py
print("IMC\n")

print("--Valores decimais devem ser digitados com (.)!--")

h=float(input("Sua altura (m): "))
m=float(input("Sua massa (kg): "))

IMC=m/h**2; IMC="{:.2f}".format(IMC)

print(f"\nSeu IMC é {IMC}\n")

File: Listas de Exercícios (C)/C/Lista 1 - Casa/5.c
#include <stdio.h>
#include <math.h>

int main ()
{
    float IMC, h, m;
    
    printf("IMC\n\n");
    
    printf("--Valores decimais devem ser digitados com (.)--\n");

    printf("Sua altura (m): "); scanf("%f",&h);
    printf("Sua massa (kg): "); scanf("%f", &m);
    
    h=pow(h, 2);
    
    IMC=m/h;
    
    printf("\nSeu IMC é %.2f\n", IMC);
}